res/controllers/Hercules-DJ-Console-Mk1-hid-scripts.js

```diff
@@ -84,20 +84,28 @@ HerculesMk1Hid.init = function() {
     //
 
     c.capture("pitchbend_down", "all", function(g, e, v) {
-        if (engine.getValue(g, "play") == 0) {
-            engine.setValue(g, "back", v > 0 ? 1 : 0);
-        }
-        else if (v > 0) {
-            engine.setValue(g, "jog", -3);
+        if (engine.getSetting("pitchBendAsJogWheel")) {
+            if (engine.getValue(g, "play") == 0) {
+                engine.setValue(g, "back", v > 0 ? 1 : 0);
+            }
+            else if (v > 0) {
+                engine.setValue(g, "jog", -3);
+            }
+        } else {
+            script.toggleControl(g, "pitch_down");
```

Comment:
using `toggleControl` has the problem that in case a message gets lost, the assumed state of the controller and the device would become disconnected. Please do `engine.setParameter(g, "pitch_down", v > 0);` instead.

Reply:
Oh right, that makes perfect sense, I was forgetting that this would obviously hit the "on" and "off" states as well and probably happen twice. Done!